src/provider/TwapV3.sol

```diff
@@ -112,7 +112,7 @@ contract TwapV3 is OracleSource, AccessControl {
 
         // tick(imprecise as it's an integer) to price
         sqrtPriceX96 =
-            TickMath.getSqrtRatioAtTick(int24((tickCumulatives[1] - tickCumulatives[0]) / int56(uint56(twapInterval))));
+            TickMath.getSqrtPriceAtTick(int24((tickCumulatives[1] - tickCumulatives[0]) / int56(uint56(twapInterval))));
```

Comment:
This seems like a big change doesn't it?

Reply:
I updated the version. They just changed the same, but it does the same thing.